### PR TITLE
Update sorted column background color

### LINKED ISSUE #448: Tabular Data: Update style of column that is actively being sorted
The original design spec called for the column that was actively being sorted to feature bold text. (as an additional affordance that the table is being sorted by the selected column)

By default, the revenue table should be sorted by date - however as it's currently built, the date remains bold regardless of whether or not you're sorting by that column. 

Rather than changing the weight of the font (which would impact the layout), I'd prefer to update the background color to `$core-light-gray-100` for cells in a column that is being sorted. 

screen shot 2018-09-18 at 3 03 04 pm

Hovering over a row would override the sorting style with the existing hover style:

screen shot 2018-09-18 at 3 04 48 pm

### PR DESCRIPTION
**Pull request by joshuatf**
Fixes #448 

Update column color in tables to light gray when sorted.  Hover color should override this color.

### Screenshots

screen shot 2018-10-18 at 9 19 12 am
screen shot 2018-10-18 at 9 19 19 am

### Detailed test instructions:

1.  Visit `/wp-admin/admin.php?page=wc-admin#/analytics/revenue`
2.  Click on any column header to sort that column
3.  Confirm that background-color of the sorted column is light gray
4.  Confirm that hovering the column overrides the sorted color with the hovered row color

### CODE REVIEW

**Comment by joshuatf** [comment]
@LevinMedia Let me know your thoughts on how this looks with that smaller clickable area or if we should modify.

**Review by Aljullu — APPROVED** [review]
Works as expected and looks great!

LGTM

File: client/components/table/style.scss [hunk 1]
```diff
@@ -80,6 +80,9 @@
 	tr:hover,
 	tr:focus-within {
 		background-color: $core-grey-light-200;
+		td {
+			background: transparent;
```
Comment (belcherj) [inline client/components/table/style.scss]: Noice.

**Review by belcherj — APPROVED** [review]
LGTM.  Nice job on that override.

**Pull request merged**